src/main/java/org/mifos/processor/bulk/HealthCheck.java

```diff
@@ -68,17 +63,17 @@ public void configure() {
                     CsvSchema schema = CsvSchema.emptySchema().withHeader();
                     MappingIterator<Transaction> readValues = csvMapper.readerWithSchemaFor(Transaction.class).with(schema).readValues(csvFile);
 
-                    while (readValues.hasNext()) {
+                    /*while (readValues.hasNext()) {
```

Comment:
this will be removed in upcoming PRs, since this PR scope is to only add empty Worker and disabled Kafka, so commented out related code in the entire code base.